services/tenant-ui/frontend/src/views/innkeeper/InnkeeperTenants.vue

```diff
@@ -4,4 +4,7 @@
 
 <script setup lang="ts">
 import Tenants from '@/components/innkeeper/tenants/Tenants.vue';
+
+// Override the page title. KISS.
+document.title = 'Traction Innkeeper Console';
```

Comment:
Is this needed as well as the router guard one above? Repeating the same string.

Reply:
No... totally not. Thought I removed it.
Good catch @loneil